app.js

```diff
@@ -50,4 +50,6 @@ app.use(function (err, req, res, next) {
     res.render('error');
 });
 
+app.listen(3000);
```

Comment:
remove this, we don't need this, we r using the /bin/www file this task

Reply:
I was just using it to check locally using the browser if the requests are being served. I will remove it once i finish writing up all the requests

Reply:
and, run app using command 'npm start'. do not run the app.js file

Reply:
Okay